gsk4/Gir.toml

```diff
@@ -72,86 +70,27 @@ manual = [
 name = "Gsk.*"
 status = "generate"
 
-[[object]]
-name = "Gsk.BorderNode"
-status = "generate"
-manual_traits = ["BorderNodeManualExt"]
-    [[object.function]]
-    name = "get_widths"
-    # gir fails to generate a fixed-size=4 array
-    manual = true
-
 [[object]]
 name = "Gsk.BroadwayRenderer"
 status = "generate"
 cfg_condition = "feature = \"broadway\""
 
-[[object]]
-name = "Gsk.ConicGradientNode"
-status = "generate"
-    [[object.function]]
-    name = "new"
-    # uses &[&ColorStop] instead of a &[ColorStop]
-    manual = true
-
-
 [[object]]
 name = "Gsk.GLShader"
 status = "generate"
 generate_builder = true
 
 [[object]]
-name = "Gsk.LinearGradientNode"
-status = "generate"
-    [[object.function]]
-    name = "new"
-    # uses &[&ColorStop] instead of a &[ColorStop]
-    manual = true
-
-[[object]]
-name = "Gsk.RadialGradientNode"
-status = "generate"
-    [[object.function]]
-    name = "new"
-    # uses &[&ColorStop] instead of a &[ColorStop]
-    manual = true
-
-[[object]]
-name = "Gsk.RenderNode"
-status = "generate"
-    [[object.function]]
-    name = "draw"
-        [[object.function.parameter]]
-            name = "cr"
-            const = true
-
-[[object]]
-name = "Gsk.RepeatingLinearGradientNode"
-status = "generate"
-    [[object.function]]
-    name = "new"
-    # uses &[&ColorStop] instead of a &[ColorStop]
-    manual = true
-
-[[object]]
-name = "Gsk.RepeatingRadialGradientNode"
-status = "generate"
-    [[object.function]]
-    name = "new"
-    # uses &[&ColorStop] instead of a &[ColorStop]
-    manual = true
-
-
-[[object]]
-name = "Gsk.ShadowNode"
+name = "Gsk.Renderer"
 status = "generate"
+manual_traits = ["RendererManualExt"]
     [[object.function]]
-    name = "new"
-    # uses &[&Shadow] instead of a &[Shadow]
+    name = "render"
+    # gir 
```

Comment:
please make this more descriptive of the reason why it's manual, same for `render_texture`

Reply:
done